core/src/main/java/io/substrait/relation/RelCopyOnWriteVisitor.java

```diff
@@ -377,14 +377,8 @@ protected Optional<ConsistentPartitionWindow.WindowRelFunctionInvocation> visitW
 
     return Optional.of(
         ConsistentPartitionWindow.WindowRelFunctionInvocation.builder()
+            .from(windowRelFunctionInvocation)
```

Comment:
Using from here both:
* saves use some lines of code
* prevents issues if we ever add fields to the `ConsistentPartitionWindow` (as we would then need to make sure we updated this code to manually copy over those fields)